### ISSUE TITLE
204 No Content

### ISSUE THREAD

**Issue by jankeesvw** [rzane/rspec-raml#3]
Hi, I'm using rspec-raml in my current project, it works great! 👍 

Today I was adding `204 No Content` to the RAML but I can't get it to pass. Do you have an example on how to get this working with RSpec?

I'm trying this:

```yaml
/users:
  securedBy:
    - basic
  /{externalUserId}/follows/show/{showId}:
    delete:
      description: Stop following a user
      responses:
        204:
```

**Comment by rzane** [comment]
No, unfortunately, I don't. I think the problem is here: https://github.com/rzane/rspec-raml/blob/master/lib/rspec/raml/matchers/match_raml_body.rb#L46.

It's just parsing no matter what, which is definitely not right. Any interest in submitting a PR?
